### ISSUE TITLE
Will the code and the training data be open?

### ISSUE THREAD

**Issue by ZihaoZheng98** [mantra-inc/open-mantra-dataset#1]

**Comment by rhina** [comment]
Sorry, the code will not be made public.
The training data is also not open because of copyright issue.
Let us know if you have any question about the implementation details of our models and training data construction.

**Comment by ZihaoZheng98** [comment]
What a pitty , without training data and code , it is too hard to follow this work
